guides/common/modules/proc_creating-compute-profiles.adoc

```diff
@@ -13,8 +13,12 @@ A default installation of {ProjectName} contains three predefined profiles:
 
 You can apply compute profiles to all supported compute resources:
 
+ifdef::provisioning,provisioning-cloud[]
```

Comment:
Are these flags used during the documentation generation?
What's the reasoning for not including these blocks in the end docs?

Reply:
There will be a third guide for provisioning cloud instances.